### ISSUE TITLE
Scatter over Image

### ISSUE THREAD

**Issue by ifollowbenoit** [bloomberg/bqplot#791]
Hi,

I'm currently trying to do a scatter over an image. But my image is not interactive, i can scatter plot around it and drag the point plotted over the image but whenever i try to plot over it directly nothing happen and I can't even have a callback when my image is clicked. Ideally I would like to update my scatter plot with the x and y position  clicked over the image.

Do you have any idea of how to proceed?
Thank you in advance

```
from bqplot import *
from IPython.display import display

import ipywidgets as widgets
import os

image_path = os.path.abspath('Map/mapv2_showroom.jpeg')

with open(image_path, 'rb') as f:
    raw_image = f.read()
ipyimage = widgets.Image(value=raw_image, format='jpg', disabled=True)
scales = {'x': LinearScale(min=-1, max=2), 'y': LinearScale(min=-0.5, max=2)}
image = Image(image=ipyimage, scales=scales)
size = 100
np.random.seed(0)
x_data = range(size)
y_data = np.cumsum(np.random.randn(size) * 100.0)


sc_x = LinearScale()
sc_y = LinearScale()

scat = Scatter(x=x_data[:10], y=y_data[:10], scales={'x': sc_x, 'y': sc_y}, default_colors=['blue'],
               enable_move=True)

ax_x = Axis(scale=sc_x)
ax_y = Axis(scale=sc_y, tick_format='0.2f', orientation='vertical')
scat.interactions = {'click': 'add'}

def callback(_, target):
    print(target)
    
image.on_click(callback)
image.interactions= {'click': 'add'}

fig = Figure(marks=[image, scat], axes=[ax_x, ax_y])

with scat.hold_sync():
    scat.enable_move = True
    scat.update_on_move = True
    scat.enable_add = True

image.x = [-1, 2]
image.y = [-.5, 2]

display(fig)
```

**Comment by rmenegaux** [comment]
Hi @ifollowbenoit. This feature has been very recently added in master (PR #692). You can pull from master and take a look at the Image example notebook to check out how it works.

**Comment by ifollowbenoit** [comment]
Thank you very much. I tried to run the following examples with the last version and it doesn't seems to be working anymore.

https://github.com/bloomberg/bqplot/blob/master/examples/Marks/Object%20Model/Image.ipynb

In fact every kind plots work fine except for the  marks=[image] plot, could you fix this issue please? 

Thank you

**Comment by rmenegaux** [comment]
Yes indeed that is a bug. I've issued a fix in #792 it should be merged soon.

**Comment by justinblaber** [comment]
> Hi @ifollowbenoit. This feature has been very recently added in master (PR #692). You can pull from master and take a look at the Image example notebook to check out how it works.

Can you link the example notebook?

**Comment by justinblaber** [comment]
> > Hi @ifollowbenoit. This feature has been very recently added in master (PR #692). You can pull from master and take a look at the Image example notebook to check out how it works.
> 
> Can you link the example notebook?

Nvm I guess it's here:

https://github.com/bloomberg/bqplot/blob/master/examples/Marks/Object%20Model/Image.ipynb

Also, would anyone happen to know the easiest way to just plot a draggble point on an image with a callback whenever it's moved? I'd like to be able to update another plot as the point is moved... if there's an example in a notebook somewhere that would be greatly appreciated...
